inyoka/portal/tasks.py

```diff
@@ -78,3 +79,25 @@ def query_counter_task(cache_key, sql):
     cursor = connection.cursor()
     cursor.execute(sql)
     cache.set(cache_key, cursor.fetchone()[0])
+
+
+@shared_task
+def clean_privmsg_folders():
+    """
+    Remove all messages in private message folders (except 'archive')
+    after end of cache duration according to settings
+    has been reached.
+    """
+    privmsgs_trash = PrivateMessageEntry.objects.filter(
+        folder="2",
```

Comment:
i would suggest to remove magic numbers (and instead use a constant in the model f.e.)